<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>463</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>463</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowserUi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>302</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSx">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSy">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起始X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起始Y</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxEx">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxEy">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>终点Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>终点X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetOk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>应用设置</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPrint">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>161</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>初始设置</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>textBrowserUi</zorder>
   <zorder>pushButtonFile</zorder>
   <zorder>spinBoxSx</zorder>
   <zorder>spinBoxSy</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>spinBoxEx</zorder>
   <zorder>spinBoxEy</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButtonSetOk</zorder>
   <zorder>textBrowserPrint</zorder>
   <zorder>pushButtonStart</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
